### ISSUE TITLE
Dyre config doesn't build

### ISSUE THREAD

**Issue by sgraf812** [yi-editor/yi#906]
I was looking for use cases of dyre and after not having been able to get a simple example to run with `stack` (frankly, invoking GHC on a single file is hardly enough for anything but the most trivial config) for myself, I was looking into how `yi` did things. Turns out I can't build it with `dyre` either, due to related errors (Edit: I used `example-configs/yi-vim-vty-dynamic`):

```
/Users/sgraf/.config/yi/yi.hs:1:8:                                                                   
   Could not find module ‘Yi.Config.Simple’                                                         
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:2:8:                                                                   
   Could not find module ‘Yi.Config.Default’                                                        
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:3:8:                                                                   
   Could not find module ‘Yi.Config.Default.Vty’                                                    
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:4:8:                                                                   
   Could not find module ‘Yi.Config.Default.Vim’                                                    
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:5:8:                                                                   
   Could not find module ‘Yi.Config.Default.HaskellMode’                                            
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:6:8:                                                                   
   Could not find module ‘Yi.Config.Default.JavaScriptMode’                                         
   Use -v to see a list of the files searched for.                                                  
                                                                                                    
/Users/sgraf/.config/yi/yi.hs:7:8:                                                                   
   Could not find module ‘Yi.Config.Default.MiscModes’                                              
   Use -v to see a list of the files searched for.  
```

Note that I installed `yi` via `stack`, so the problem might be that there are no `yi` modules in the package path, but I'm not sure.

Given how controversial `dyre` seems to be anyway from the issues I read through, I thought I also (deploringly) recommend to discontinue supporting it. The difference between starting with a `stack new` template and recompiling that as needed and using `dyre` isn't big. There's also the possibility to use `stack` as a script interpreter on the config file if quick turnaround time for adjusting the config is needed.

**Comment by jhance** [comment]
`stack exec yi --package yi-vim-vty-dynamic`?

**Comment by sgraf812** [comment]
Uhmm... I don't follow. Maybe it's just me not having read the config section more thoroughly.

~~Where should I execute that line? It can't find `yi-vim-vty-dynamic` if I execute it within the `example-configs/yi-vim-vty-dynamic` even if I `stack install` prior to that.~~ OK, it's `yi-config`. What I did so far:

- Clone the repo
- `stack install` at the root
- copy the contents of `example-configs/yi-vim-vty-dynamic` to `~/.config/yi/`
- Execute `yi`

That doesn't build because of missing dependencies.

What I tried just now:
- modify package locations in the `stack.yaml` of `example-configs/yi-vim-vty-dynamic` to point to GitHub
- copy the contents of `example-configs/yi-vim-vty-dynamic` to `~/.config/yi/`
- `stack exec yi --package yi-config` within `example-configs/yi-vim-vty-dynamic`

The stack build succeeds, but the rebuild breaks again because of the same missing imports.
